lasagne/layers/recurrent.py

```diff
@@ -106,8 +106,13 @@ class CustomRecurrentLayer(MergeLayer):
     input_to_hidden : :class:`lasagne.layers.Layer`
         :class:`lasagne.layers.Layer` instance which connects input to the
         hidden state (:math:`f_i`).  This layer may be connected to a chain of
-        layers, which must end in a :class:`lasagne.layers.InputLayer` with the
-        same input shape as `incoming`.
+        layers, which must end in a :class:`lasagne.layers.InputLayer`.
+        ``input_to_hidden.input_shape[1:]`` should always be the same as
```

Comment:
That needs to be formulated differently... `input_to_hidden.input_shape[1:]` is the input shape of the last layer of `input_to_hidden`, while we're interested in the input shape of the aforementioned `InputLayer`.